src/select/styled-components.js

```diff
@@ -37,6 +37,9 @@ export const Input = styled('input', props => {
     flexGrow: '1',
     paddingTop: scale300,
     paddingBottom: scale300,
+    ':hover': {
+      cursor: 'pointer',
+    },
```

Comment:
Did we add this back for some reason? seems like the `:hover` shouldn't be needed if we rebase with master.

Reply:
ok, now i see it in master. removed again